odoo/sql_db.py

```diff
@@ -464,57 +411,24 @@ def _close(self, leak=False):
             self._cnx.leaked = True
         else:
             chosen_template = tools.config['db_template']
-            templates_list = tuple(set(['template0', 'template1', 'postgres', chosen_template]))
+            templates_list = {'template0', 'template1', 'postgres', chosen_template}
             keep_in_pool = self.dbname not in templates_list
```

Comment:
Making a `set` for that only looks way overkill.  It will never gonna be faster than
```suggestion
            keep_in_pool = self.dbname not in ('template0', 'template1', 'postgres', chosen_template)
```

Reply:
Yes, Agree.